Angular/angular.md

```diff
@@ -0,0 +1,45 @@
+
+# Angular
+
+## Motivation
+
+- Angular is a popular Javascript Framework that is being used to build Applications for Web, Desktop and Mobile Devices. 
+- Launched by Google in 2010 as AngularJS, the subsequent versions of AngularJS are simply called Angular and involves the usage of Typescript which is a superset of Javascript. 
+- Typescript is compatible with Javascript and compiles to Javascript which is rendered on the Browser.
+- Angular is similar to React.js and Vue which are being used for creating Front-End User Interfaces.
+- What makes Angular quite intutive and easy to learn is that, Angular uses HTML to define the User-Interface and hence we don't need to write the Front-End Interface in Javascript. 
+- HTML is a Markup language and follows a well-defined layout where we can bind the data using Angular and readily render an interface with minimal effort.
+
+## What problems does it solve?
+One of the greatest advantages with Angular is that, it is Cross-Platform and Modular. Angular can be used for developing Web Application along with Progressive Web Applications, Desktop Applications and Native Mobile Applications. Angular is also modular as the whole code is divided into "modules" which consists of components, pipes and directives coupled together. These modules are reusable and helps implement "Lazy Loading" which means that one or more features of the application can be loaded anytime.
+
+### Why is it recommended?
+Angular is recommended to learn because of the Model View Controller (MVC) Architecture that it follows which allows the developers to do more while written lesser lines of code. Angular is also pre-built in many existing frameworks like Ionic and Wejmo and hence allows to add attractive User-Interfaces. Angular also supports Single Page Applications which are updated dynamically by communicating with the Backend Server. This makes Angular worth learning.
```

Comment:
Can you condense this entire section? It should be a few sentences, not a few paragraphs.

Most good frameworks are modular so this isn't really a reason to choose Angular. This might be difficult to do if you don't have a lot of expertise in multiple frameworks, so I'd suggest asking more experienced Angular fans why they prefer it.